test/unit-minimal/main.tf

```diff
@@ -0,0 +1,54 @@
+module "test" {
+  source = "../.."
+
+  # add all required arguments
+
+  cloud_function = "test"
```

Comment:
Please add only required arguments in the minimal test. Basically anything that does not have a default value.